### PR TITLE
chore: Gitpodify melt-ui

### PR DESCRIPTION
**Pull request by KapadiaNaitik**
Made melt-ui gitpod ready so contirbutors can easily set it up in one click and get it up and running super quick.
Helps to review changes made in PR with ease.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ enhance the experience of building with Svelte and improve the ecosystem for eve
 
 Check out our [Contributing guide](./CONTRIBUTING.md) to learn more.
 
+
+## Contribute from a browser IDE
```
Comment (TGlide) [inline README.md:64]: I'd add this to CONTRIBUTING.md instead

**Review by TGlide — COMMENTED** [review]
Thank you for the PR!

File: README.md (hunk 1 above)
Comment (KapadiaNaitik) [reply README.md]: @TGlide did the same

**Comment by KapadiaNaitik** [comment]
@TGlide I have added it to contributing.md. Please review it and let me know.

**Review by TGlide — APPROVED** [review]
Awesome, thank you! 🎉

**Pull request merged**